src/tlo/methods/breast_cancer.py

```diff
@@ -646,6 +647,9 @@ def __init__(self, module, person_id):
         self.TREATMENT_ID = "BreastCancer_Investigation"
         self.EXPECTED_APPT_FOOTPRINT = self.make_appt_footprint({"Over5OPD": 1, "Mammography": 1})
         self.ACCEPTED_FACILITY_LEVEL = '3'  # Mammography only available at level 3 and above.
+        # TODO: what this means, should be the mammography done within this event, or the biopsy, or both?
+        self.EQUIPMENT = {'Slice Master sample processing Unit', 'Paraffin Dispense', 'Whatever used with biopsy',
+                          'Mammography maybe?'}  # biopsy and ?mammography always performed with this HSI
```

Comment:
let's take mammography out

Reply:
@EvaJanouskova @tbhallett 
Just trying to look back at this BreastCancer module and understand the code.  On line 669 I think we should have this        

df.at[person_id, 'brc_breast_lump_discernible_investigated'] = True

instead of 

df.brc_breast_lump_discernible_investigated = True

Reply:
I think you are right. @tbhallett, could you confirm this? Should this be changed in small separate PR?

Reply:
Hi @tbhallett, Should we do this change here or in a separate PR?

Reply:
We can do it here as it so small.